src/borg/compress.pyx

```diff
@@ -344,6 +406,36 @@ class CompressionSpec:
             else:
                 raise ValueError
             self.level = level
+        elif self.name in ('zstd',):
+            if count < 2:
+                level = 5  # default compression level for zstd
+                threads = 1 # default 1 thread
+            elif count == 2:
+                threads = 1 # default 1 thread
+                level = int(values[1])
+                if not 0 <= level <= 22:
+                    raise ValueError
+            elif count == 3:
+                maxcpu = mp.cpu_count() #get cpu count
+                level = int(values[1])
+                if not 0 <= level <= 22:
+                    raise ValueError
+
+                threads = int(values[2])
+
+                if threads == 0:
+                    threads=maxcpu # use maximum avaliable cpu's
```

Comment:
pep8: blanks around operators, so it is " = ". also fix typos: "available" and "CPUs"